packages/wdio-allure-reporter/README.md

```diff
@@ -115,6 +124,20 @@ Given('I include feature and story name', () => {
 })
 ```
 
+#### Custom steps
+
+`step` method simplifies dealing with steps because each step present as an async function with any content inside.
+The first argument of the function is the current step, that has most of the allure API methods (such as `label`, `epic`, `attach` etc):
+
+```js
+allureReporter.step('my step name', async (s1) => {
+    s1.label('foo', 'bar')
+    await s1.step('my child step name', async (s2) => {
+        // you ca
```

Comment:
It seems that a valuable comment has been missed.

Reply:
:+1: